### ISSUE TITLE
Add documentation on how to use redux-dag-history and dag-history-component

### ISSUE THREAD

**Issue by jimmy-guo** [Microsoft/redux-dag-history#37]
Looking like this would be a good fit in one of my projects, but just starting out with Redux, it is not obvious to me how I should use redux-dag-history and dag-history-component. Would it be possible to add some documentation?

**Comment by darthtrevino** [comment]
You're absolutely right about the docs, I'll put some work in there. You can run the example by executing:
`yarn install` then `yarn start`. It should start up a simple app at localhost:8080 that demonstrates usage of the components and middleware.
